src/Autonomous.h

```diff
@@ -69,6 +71,7 @@ class Autonomous : rclcpp::Node
 	ControlState control_state;
 	int time_since_plan;
 	plan_t plan;
+	std::future<plan_t> pending_plan;
```

Comment:
Should we explicitly initialize this in the constructor?

Reply:
I think it only has a default constructor and a move constructor, so it shouldn't be strictly necessary, but we can do it to be explicit.